src/base/ReactorStructs.sol

```diff
@@ -9,14 +9,12 @@ import {IRelayOrderReactor} from "../interfaces/IRelayOrderReactor.sol";
 struct RelayOrder {
     // Generic order info
     OrderInfo info;
-    // Token info for the onchain trade and the payout to fillers
-    Input[] inputs;
-    // The time at which the inputs start decaying
-    uint256 decayStartTime;
-    // The time at which price becomes static
-    uint256 decayEndTime;
-    // ecnoded actions to execute onchain
-    bytes[] actions;
+    // Token info for onchain actions
+    Input input;
+    // The fee offered for the order
+    FeeEscalator fee;
+    // ecnoded data to execute onchain
+    bytes actions;
```

Comment:
nit: mabye rename / update comment here to specify this is strictly for UniversalRouter? i.e. `universalRouterCalldata`?

Reply:
sgtm!

Reply:
oo good idea, I'll do this in a follow up if ok